dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -263,6 +263,21 @@
     E6003( "Could not patch object with id {0}." ),
     E6004( "Attribute `{0}` has invalid GeoJson value." ),
     E6005( "Attribute `{0}` has unsupported GeoJson value." ),
+    E6006( "AttributeValue `{0}` is an invalid Integer value." ),
```

Comment:
I think we can use lower-case for the value types, like "integer" and "positive integer value".

Reply:
ok @larshelge , i will update those in another PR